packages/create-discord-bot/template/TypeScript/src/index.ts

```diff
@@ -1,18 +1,24 @@
 import process from 'node:process';
 import { URL } from 'node:url';
 import { Client, GatewayIntentBits } from 'discord.js';
-import { loadCommands, loadEvents } from './util/loaders.[REPLACE_IMPORT_EXT]';
-import { registerEvents } from './util/registerEvents.[REPLACE_IMPORT_EXT]';
+import { loadEvents } from './util/loaders.[REPLACE_IMPORT_EXT]';
 
 // Initialize the client
 const client = new Client({ intents: [GatewayIntentBits.Guilds] });
 
 // Load the events and commands
 const events = await loadEvents(new URL('events/', import.meta.url));
-const commands = await loadCommands(new URL('commands/', import.meta.url));
 
 // Register the event handlers
-registerEvents(commands, events, client);
+for (const event of events) {
+	client[event.once ? 'once' : 'on'](event.name, async (...args) => {
+		try {
+			event.execute(...args);
```

Comment:
if `execute` is async this try block doesn't do anything
applies to other places as well